src/lib/components/text-field/index.tsx

```diff
@@ -171,45 +176,29 @@ export const TextField: FC<PropsWithRef<TextFieldProps>> = forwardRef(
               }}
               {...props}
             />
-            {Bottom && (
-              <Typography
-                size="small"
-                variant="body"
-                color="onSurface"
-                textAlign={props.textAlign}
-              >
-                {Bottom}
-              </Typography>
-            )}
           </Box>
-          {(variant == 'error' && (
-            <ErrorSVG
-              width="100%"
-              height="100%"
-              maxWidth="1.25rem"
-              maxHeight="1.25rem"
-            />
-          )) ||
-            (valid && variant == 'valid' && (
-              <TickSVG
-                width="100%"
-                height="100%"
-                maxWidth="1.25rem"
-                maxHeight="1.25rem"
-              />
-            )) ||
-            SuffixIcon}
-          {Suffix}
+          <Box
+            p="xs"
+            px=".625rem"
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+          >
+            {Suffix}
+          </Box>
         </Motion>
-        {statusColor && (
+        {statusColor ? (
           <Typography variant="body" mt="2xs" size="small">
             {error || valid}
           </Typography>
+        ) : (
+          <Typography variant="body" mt="2xs" size="small">
+            {supportingText}
```

Comment:
@KipandaJr supportingText, is different from the "error" and "valid" and other statuses, because sometimes, we'll need supportingText, without any status. So supportingText is a normal text under the input